client/tests/setup/SessionProviderMock.tsx

```diff
@@ -0,0 +1,7 @@
+// Needed to put this in a tsx file since .ts files are not allowed to have jsx
```

Comment:
If you haven't already tried this, can we put all the mocks in a tsx file to keep them together?